apps/core/scenes/search/PlacePicker/PlacePickerContent.js

```diff
@@ -0,0 +1,116 @@
+// @flow
+
+// TODO this is just the temporary Picker will be replaced by the OptionPicker form the 'universal-components'
+
+import { graphql, createRefetchContainer } from '@kiwicom/margarita-relay';
+import { type RelayRefetchProp } from 'react-relay'; // eslint-disable-line no-restricted-imports
+import * as React from 'react';
+import { View, ScrollView } from 'react-native';
+import {
+  Text,
+  TextInput,
+  Icon,
+  StyleSheet,
+} from '@kiwicom/universal-components';
+
+import type { PlacePickerContent_locations as PlacePickerContentType } from './__generated__/PlacePickerContent_locations.graphql.js';
+
+type Props = {|
+  +locations: ?PlacePickerContentType,
+  +type: string,
+  +relay: RelayRefetchProp,
+|};
+
+type State = {|
+  text: string,
+|};
+
+class PlacePicker extends React.Component<Props, State> {
+  state = {
+    text: '',
+  };
+
+  handleChangeText = (text: string) => {
+    this.setState({ text });
+    this.props.relay.refetch({ input: { term: text } });
+  };
+
+  getLabel = type => {
+    if (type === 'DESTINATION') {
+      return 'To';
+    } else if (type === 'ORIGIN') {
+      return 'From';
+    }
+    return null;
+  };
+
+  render() {
+    const locations = this.props.locations?.locationsByTerm?.edges ?? [];
+
+    return (
+      <View style={styles.container}>
+        <View>
+          <Text weight="bold">{this.getLabel(this.props.type)}:</Text>
+          <TextInput
+            placeholder="write a place..."
+            onChangeText={this.handleChangeText}
+            value={this.state.text}
+            prefix={<Icon name="search" />}
+          />
+        </View>
+        <ScrollView style={styles.list}>
+          {locations.map(location => (
+            <View style={styles.wrapper} key={location?.node?.id}>
+              <Icon name="location" />
+              <Text style={styles.listItem} key={location?.node?.id}>
```

Comment:
Do you need `key` here? Its parent has it

Reply:
done